<ui version="4.0" >
 <class>qdArchiveFileProperties</class>
 <widget class="QDialog" name="qdArchiveFileProperties" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>236</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Archive file properties</string>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>File:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>30</y>
     <width>72</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Unpacked size:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>50</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Packed size:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>90</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>CRC:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>70</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>Method:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>110</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Date/time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>130</y>
     <width>52</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Attributes:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qpbClose" >
   <property name="geometry" >
    <rect>
     <x>150</x>
     <y>160</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>&amp;Close</string>
   </property>
  </widget>
  <widget class="QLabel" name="qlFileName" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="qlUnpackedSize" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="qlPackedSize" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>50</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="qlMethod" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>70</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="qlCRC" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="qlDateTime" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>110</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="qlAttributes" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>130</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>qpbClose</sender>
   <signal>clicked()</signal>
   <receiver>qdArchiveFileProperties</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>87</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>95</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
